clap_complete/tests/testsuite/common.rs

```diff
@@ -252,6 +252,19 @@ pub fn value_hint_command(name: &'static str) -> clap::Command {
         )
 }
 
+pub fn value_help_command(name: &'static str) -> clap::Command {
```

Comment:
Can you add this to all of the shells?  That doesn't mean you need to make sure it works properly with all of them

Reply:
nm, going to suggest something else